itest/client_harness.go

```diff
@@ -19,36 +19,39 @@ type clientHarness struct {
 	grpcConn   *grpc.ClientConn
 	clientConn mockrpc.MockServiceClient
 
-	cancel func()
-}
+	password     []byte
```

Comment:
should we call this `pairingPhrase` everywhere to make it a bit more clear what it actually is? I know I used `password` in a few places of the code before, but if we're touching this anyway we might also rename things.

Reply:
cool. Are you guys happy with the name `passphraseEntropy` here as i have done in the last commit of the pr? seemed important to just signal in the name that it is not in its mnemonic form here